### PR TITLE
Make parts of sdk-ui and sdk-backend-spi public

### PR DESCRIPTION
**Pull request by kandl**
- Parts that are alpha or beta for a long time without significant changes are now public
- Also split Execution live examples to tabs and add example for ExecuteInsight / useInsightDataView

JIRA: RAIL-4082

---

Supported PR commands:

| Command                  | Description             |
| ------------------------ | ----------------------- |
| `ok to test`             | Re-run standard checks  |
| `extended test`          | BackstopJS tests        |
| `extended check sonar`   | SonarQube tests         |
| `extended check cypress` | Cypress E2E tests       |
| `extended check plugins` | Dashboard plugins tests |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `check-extended-cypress` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

File: examples/sdk-examples/src/examples/execution/UseInsightDataView.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import UseInsightDataView from "./UseInsightDataViewSrc";
+import UseInsightDataViewSRC from "./UseInsightDataViewSrc?raw";
+import UseInsightDataViewSRCJS from "./UseInsightDataViewSrc?rawJS";
+
+const UseInsightDataViewHook = (): JSX.Element => (
+    <div>
+        <h1>useInsightDataView</h1>
+        <p>
+            useInsightDataView hook is an alternative to ExecuteInsight component. The advantage of this hook
+            over ExecuteInsight component is that it do not depend on rendering, and it
```
Comment (no23reason) [inline examples/sdk-examples/src/examples/execution/UseInsightDataView.tsx:16]: Typo: does not depend

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/7366/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/9215/ : SUCCESS in 16m 34s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR2376

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/2129/ : SUCCESS in 1m 06s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/2414/ : SUCCESS in 20s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/1233/ : SUCCESS in 15m 00s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/9219/ : SUCCESS in 18m 23s

**Pull request merged**